### PR TITLE
Added University of Melbourne site

### PR DESCRIPTION
**Pull request by whiskerwind**
Added new website to entry directory

### CODE REVIEW

File: entries/u/unimelb.edu.au.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "University of Melbourne": {
+    "domain": "unimelb.edu.au",
+    "img": "/img/unimelb.edu.au.svg",
```
Comment (phallobst) [inline entries/u/unimelb.edu.au.json]:
```suggestion
```

File: entries/u/unimelb.edu.au.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "tfa": [
+      "custom-software"
+    ],
+
```
Comment (phallobst) [inline entries/u/unimelb.edu.au.json]:
```suggestion
    "custom-software": [
      "Okta Verify"
    ],
```

File: entries/u/unimelb.edu.au.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "documentation": "https://studentit.unimelb.edu.au/cybersecurity/multifactor-authentication-mfa",
+     "notes": "2FA can only be enabled using the Okta Verify app",
```
Comment (phallobst) [inline entries/u/unimelb.edu.au.json]:
```suggestion
```

File: entries/u/unimelb.edu.au.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "keywords": [
+      "education"
```
Comment (phallobst) [inline entries/u/unimelb.edu.au.json]:
```suggestion
      "universities"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Your SVG file needs to be minimized. I recommend you use [SVGOM](https://jakearchibald.github.io/svgomg/)

**Comment by phallobst** [comment]
74K for an SVG logo is quite hefty. I managed to get it down to 40K with some optimization, but that's still a bit much. If there is no alternative logo available, I reckon we should go with a PNG in this case. It's a bit mushy at 32x32px, but better than nothing:
unimelb edu au

This will also require the following line:
```
    "img": "unimelb.edu.au.png",
```

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**